### PR TITLE
Troubleshooting a cluster in error

### PR DESCRIPTION
**Pull request by everett-toews**
@carolynvs @kenperkins @ktbartholomew @jamiehannaford @coreypobrien

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-1efc92b254/)

File: _troubleshooting/2017-01-27-cluster-in-error.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: Troubleshooting a cluster in error on Carina
+docker-versions:
+  - 1.11.2
+kubernetes-versions:
+  - 1.4.5
+topics:
+  - troubleshooting
+---
+
+When you initiate a task on a cluster, like create or resize, Carina performs that task asynchronously. That task may cause a cluster to go into the `error` status. The Carina team is automatically notified about every cluster that goes into the `error` status and someone is assigned to investigate. Our aim is to understand the root cause of these errors and mitigate them so you have a robust and stable platform to build on.
```
Comment (ktbartholomew) [inline _troubleshooting/2017-01-27-cluster-in-error.md]: s/mitigate/prevent ? I think that communicates that we would like to have fewer and fewer of these errors as time goes on.

File: _troubleshooting/2017-01-27-cluster-in-error.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+However, at this time, we cannot move a cluster out of the `error` status and that cluster continues to count against your quota. Your only recourse is to delete the cluster and try again.
```
Comment (ktbartholomew) [inline _troubleshooting/2017-01-27-cluster-in-error.md]: s/only/best. sounds less bleak. :)

**Comment by carolynvs** [comment]
Often the reason people want to "resurrect" an errored cluster is because they have data they want off of it, like their database files. Can we provide either best practices to avoid that situation or tips on how to recover data off an errored cluster?

**Comment by everett-toews** [comment]
@carolynvs Good thought but we don't have a definitive doc on data store best practices that's cluster agnostic. However, whether we have such a definitive doc or not, by the time they get to this troubleshooting doc it's too late. It would be better to include or link to that kind of info in getting started guides and other tutorials.

That said, I'll include some verbiage around best practices and link to what we do have as a "do this next time" message. I'll also mention that they should still at least attempt to connect to their cluster to retrieve data.

**Comment by carolynvs** [comment]
Yup! "Do this next time so you don't lose data" would be helpful.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-89b9d4b65b/)

**Comment by carolynvs** [comment]
👍

**Pull request merged**